SparkPost/2web2ui
# AC-1252: Onboarding updates

### What Changed
 - Removed the welcome modal
 - Remove the features step in the onboarding

### How To Test
 - Create a new account
 - Verify that modal no longer shows when getting to /dashboard
 - Go to `/dashboard` and verify that the Features step is removed

### Notes
 - Added in a catch for users who may have had guide step as "Features" and would catch as "Sending"
